### PR TITLE
Config changes to reflect the default branch from master to main. Please merge to complete the default branch renaming job.

### PR DESCRIPTION
**Pull request by BillWagner**
Auto branch renaming content.

### CODE REVIEW

File: .openpublishing.publish.config.json [hunk 1]
```diff
@@ -36,38 +36,38 @@
...
-      "branch": "master",
+      "branch": "main",
       "branch_mapping": {}
     },
     {
       "path_to_root": "_csharplang",
       "url": "https://github.com/dotnet/csharplang",
       "branch": "main",
-      "include_in_build": true
+      "branch_mapping": {}
```
Comment (gewarren) [inline .openpublishing.publish.config.json:52]: This needs to be reverted - it's causing build warnings. Same for the 3 changes below this one.

**Review by gewarren — DISMISSED** [review]

File: .openpublishing.publish.config.json (hunk 1 above)
Comment (BillWagner) [inline .openpublishing.publish.config.json:52]:
```suggestion
      "include_in_build": true,
      "branch_mapping": {}
```

File: .openpublishing.publish.config.json [hunk 2]
```diff
@@ -36,38 +36,38 @@
...
     },
     {
       "path_to_root": "_csharpstandard",
       "url": "https://github.com/dotnet/csharpstandard",
       "branch": "draft-v6",
-      "include_in_build": true
+      "branch_mapping": {}
```
Comment (BillWagner) [inline .openpublishing.publish.config.json:58]:
```suggestion
      "include_in_build": true,
      "branch_mapping": {}
```

File: .openpublishing.publish.config.json [hunk 3]
```diff
@@ -36,38 +36,38 @@
...
     },
     {
       "path_to_root": "_vblang",
       "url": "https://github.com/dotnet/vblang",
       "branch": "main",
-      "include_in_build": true
+      "branch_mapping": {}
```
Comment (BillWagner) [inline .openpublishing.publish.config.json:64]:
```suggestion
      "include_in_build": true,
      "branch_mapping": {}
```

File: .openpublishing.publish.config.json [hunk 4]
```diff
@@ -36,38 +36,38 @@
...
     },
     {
       "path_to_root": "_roslyn",
       "url": "https://github.com/dotnet/roslyn",
       "branch": "main",
-      "include_in_build": true
+      "branch_mapping": {}
```
Comment (BillWagner) [inline .openpublishing.publish.config.json:70]:
```suggestion
      "include_in_build": true,
      "branch_mapping": {}
```

**Review by BillWagner — COMMENTED** [review]
Restore dependent repositories.

**Review by BillWagner — COMMENTED** [review]
Restore dependent repositories.

**Review by adegeo — APPROVED** [review]

**Pull request merged**